### PR TITLE
Add Office Hours/ Mentoring of different SIGs

### LINKED ISSUE #4409: Index sig / contributor office hours
Many sigs have adopted contributor office hours or onboarding sessions to help new contributors. These are scattered across several resources, with most of them listed in sigs.yaml / sig-list.md. 

These should be indexed and linked to from our general mentoring docs.

/sig contributor-experience
/area mentorship-planning
/milestone Next

### PR DESCRIPTION
**Pull request by championshuttler**
Signed-off-by: championshuttler <shivams2799@gmail.com>

Fixes  #4409

cc @mrbobbytables

### CODE REVIEW

**Comment by parispittman** [comment]
@championshuttler thanks so much for doing this! :) 

we don't need to include any of the regular sig meetings or anything that doesn't explicitly call out office hours or mentoring. if you can edit that, happy to merge after.

**Comment by championshuttler** [comment]
> @championshuttler thanks so much for doing this! :)
> 
> we don't need to include any of the regular sig meetings or anything that doesn't explicitly call out office hours or mentoring. if you can edit that, happy to merge after.

Hi @parispittman thanks for the quick review, updating the PR PTAL :)

**Comment by mrbobbytables** [comment]
One other thing if you don't mind, please link out to the sig meetings pages or agendas. That way it's a one stop shop for folks to view get to them 👍 

Example:
```
[kops Office Hours - Biweekly Friday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit)
```

**Comment by championshuttler** [comment]
> One other thing if you don't mind, please link out to the sig meetings pages or agendas. That way it's a one stop shop for folks to view get to them 👍
> 
> Example:
> 
> ```
> [kops Office Hours - Biweekly Friday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/12QkyL0FkNbWPcLFxxRGSPt_tNPBHbmni3YLY-lHny7E/edit)
> ```

@mrbobbytables Done

File: mentoring/programs/office-hours.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    * [Conformance office Hours - Biweekly Tuesday at 19:00 UTC](https://docs.google.com/document/d/1W31nXh9RYAb_VaYkwuPLd1hFxuRX3iU0DmaQ4lkCsX8/edit#)
+* Cluster Lifecycle - 
+    * [Cluster API (AWS implementation) office hours - Biweekly Monday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1ojR0jooQaHEuz3gGW6DxcZlsws9ZX7jHPC8walm1G3A/edit)
+    * [Cluster API office hours - Biweekly Wednesday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1fQNlqsDkvEggWFi51GVxOglL2P1Bvo2JhZlMhm2d-Co/edit)
+    * [Cluster Addons meeting - Biweekly Tuesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/10_tl_SXcFGb-2109QpcFVrdrfnVEuQ05MBrXtasB0vk/edit)
+    * [Image Builder office hours - Biweekly Wednesday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1YIOD0Nnid_0h6rKlDxcbfJaoIRNO6mQd9Or5vKRNxaU/edit)
+    * [Kubespray Office Hours - Biweekly Wednesday at 08:00 PT (Pacific Time)](https://docs.google.com/document/d/1oDI1rTwla393k6nEMkqz0RU9rUl3J1hov0kQfNcl-4o/edit)
+    * etcdadm Office Hours - Biweekly Monday at 09:00 PT (Pacific Time)
```
Comment (mrbobbytables) [inline mentoring/programs/office-hours.md:19]:
```suggestion
    * [etcdadm Office Hours - Biweekly Monday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit)
```
I was able to find their meeting notes link from their slack channel. If you'd like to do another PR, sigs.yaml could be updated to include the link to the doc:
https://docs.google.com/document/d/1b_J0oBvi9lL0gsPgTOrCw1Zlx3e7BYEuXnB3d2S15pA/edit

Similar to your other one with updating the timezones :)

File: mentoring/programs/office-hours.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+title: Office Hours/ Mentoring
+weight: 4
+description: Office hours of different SIGs
+---
+
+* API Machinery - 
+    * [Kubebuilder and Controller Runtime Meeting - Second Wednesday every month at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1Ih-2cgg1bUrLwLVTB9tADlPcVdgnuMNBGbUl4D-0TIk/edit?usp=sharing)
+* Architecture - 
+    * [Enhancements Subproject Meeting - Biweekly Tuesday at 08:30 PT (Pacific Time)](https://bit.ly/k8s-enhancements-agenda)
```
Comment (mrbobbytables) [inline mentoring/programs/office-hours.md:9]:
```suggestion
```
This one is a focused meeting and not really intended for mentoring / office hours

File: mentoring/programs/office-hours.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    * [Production Readiness Office Hours - Biweekly Wednesday at 12:00 PT (Pacific Time)](https://docs.google.com/document/d/10QkXwiZfBL7wBKHHFslooXvGDKHA75jkkgiIXuKFryM/edit?usp=sharing)
+    * [Code organization Office Hours - Biweekly Thursday at 14:00 PT (Pacific Time)](https://docs.google.com/document/d/1HtTI0rJEGP_MSf6eO87aCmx_tzpovPAAg7U2Zxwm8FE/edit#)
+    * [Conformance office Hours - Biweekly Tuesday at 19:00 UTC](https://docs.google.com/document/d/1W31nXh9RYAb_VaYkwuPLd1hFxuRX3iU0DmaQ4lkCsX8/edit#)
+* Cluster Lifecycle - 
+    * [Cluster API (AWS implementation) office hours - Biweekly Monday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1ojR0jooQaHEuz3gGW6DxcZlsws9ZX7jHPC8walm1G3A/edit)
+    * [Cluster API office hours - Biweekly Wednesday at 10:00 PT (Pacific Time)](https://docs.google.com/document/d/1fQNlqsDkvEggWFi51GVxOglL2P1Bvo2JhZlMhm2d-Co/edit)
+    * [Cluster Addons meeting - Biweekly Tuesday at 09:00 PT (Pacific Time)](https://docs.google.com/document/d/10_tl_SXcFGb-2109QpcFVrdrfnVEuQ05MBrXtasB0vk/edit)
```
Comment (mrbobbytables) [inline mentoring/programs/office-hours.md:16]:
```suggestion
```
This one too.

File: mentoring/programs/office-hours.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    * [minikube Office Hours - Biweekly Monday at 11:00 PT (Pacific Time)](https://docs.google.com/document/d/1jhfmL1gsgN39uCEgz5pW9tnIotFgHhxq2yfMK3KYE4w/edit)
+* Docs - 
+    * [Korean Team Meeting - Biweekly Thursday at 13:00 UTC](https://docs.google.com/document/d/1h5sMhBpPB5unJmBAS7KzDiPs-_eFQOu5o4UyHwMtFCA/edit)
+* Multicluster - 
+    * [Cluster Federation (KubeFed) Working Group - Biweekly Wednesday at 07:30 PT (Pacific Time)](https://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY/edit)
+* Network - 
+    * [SIG Network Meeting - Biweekly Thursday at 14:00 PT (Pacific Time)](https://docs.google.com/document/d/1_w77-zG_Xj0zYvEMfQZTQ-wPP4kXkpGD8smVtW_qqWM/edit)
+    * [Service APIs Meeting (APAC Friendly Time) - Biweekly Thursday at 16:30 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
+    * [Service APIs Meeting (EMEA Friendly Time) - Biweekly Thursday at 10:30 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
```
Comment (mrbobbytables) [inline mentoring/programs/office-hours.md:30]:
```suggestion
```
I think these are all regular meetings^^;;; but may need to confirm the service api meetings.

File: mentoring/programs/office-hours.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    * [Service APIs Office Hours - Biweekly Wednesday at 15:00 PT (Pacific Time)](https://docs.google.com/document/d/1eg-YjOHaQ7UD28htdNxBR3zufebozXKyI28cl2E11tU/edit)
+* Testing -
+    * [SIG Testing Office Hours - Biweekly Tuesday at 10:00 PT (Pacific Time)](https://bit.ly/k8s-sig-testing-notes)
+* Steering -
+    * [Public Steering Committee Meeting - Monthly Monday at 11:00 PT (Pacific Time)](https://bit.ly/k8s-steering-wd)
```
Comment (mrbobbytables) [inline mentoring/programs/office-hours.md:35]:
```suggestion
```
This one too.

File: mentoring/programs/office-hours.md (hunk 1 above)
Comment (championshuttler) [reply mentoring/programs/office-hours.md]: Sure thing will do this

**Comment by neolit123** [comment]
just to note, in SIG Cluster Lifecycle we rotate our google docs occasionally - i.e. leave old documents and introduce new ones as they can become slow to navigate if there are too many recorded events / meetings / pages, then update k/community, calendar invites.

this introduces the potential for the links that this PR adds to become outdated.
ideally, there should be a way for this information to be generated / fetched from sigs.yaml.

File: mentoring/programs/office-hours.md (hunk 1 above)
Comment (championshuttler) [reply mentoring/programs/office-hours.md]: @mrbobbytables https://github.com/kubernetes/community/pull/4986 here you go :)

**Comment by mrbobbytables** [comment]
> this introduces the potential for the links that this PR adds to become outdated.
> ideally, there should be a way for this information to be generated / fetched from sigs.yaml.

I agree there. The next iteration might be to flag the meetings for office hours / mentoring focus and then adjusting the README template to render them as a section and/or generate a new page like sig-list.md

For now though I think this is still fine to do 👍 
/lgtm
/assign @parispittman

**Comment by parispittman** [comment]
/approve

thank you!
